Main.cc

```diff
@@ -460,11 +498,15 @@ int main(int argc, const char* argv[]) {
 
         session_number = 0;
 
-        websocket_hub.onMessage(&OnMessage);
-        websocket_hub.onConnection(&OnConnect);
-        websocket_hub.onDisconnection(&OnDisconnect);
+        // Create a hub group and pass the PERMESSAGE_DEFLATE option
+        group = websocket_hub.createGroup<uWS::SERVER>(uWS::PERMESSAGE_DEFLATE);
```

Comment:
I think you can do this without creating a group explicitly, by just passing these arguments to the hub constructor directly.